orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportDataService.java

```diff
@@ -73,12 +56,11 @@ public AccountingExportDataDto createAccountingExportDataDto(AccountingSlipDto a
 
         log.debug("generated accounting export data xml successfully");
 
-        String postingDate = accountingSlip.getPostingDate();
-        LocalDate postingDateFormatted = LocalDate.parse(postingDate, DateTimeFormatter.BASIC_ISO_DATE);
+        LocalDate postingDate = accountingSlip.getPostingDate();
 
         AccountingExportDataDto exportDataDto = new AccountingExportDataDto(
                 accountingSlip.getAccountingSlipId(),
-                postingDateFormatted.toString(),
+                postingDate,
```

Comment:
Could this be just `accountingSlip.getPostingDate()` no need for variable decleration